common/djangoapps/enrollment/api.py

```diff
@@ -414,7 +416,9 @@ def _validate_course_mode(course_id, mode, is_active=None):
     """
     # If the client has requested an enrollment deactivation, we want to include expired modes
     # in the set of available modes. This allows us to unenroll users from expired modes.
-    include_expired = not is_active if is_active is not None else False
+    # This will be only checked if we did not provide "include_expired" value explicitly.
```

Comment:
I believe the comment should be 'If include_expired is set as True we should not redetermine its value.'